fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanRepaymentScheduleInstallment.java

```diff
@@ -140,6 +140,9 @@ public class LoanRepaymentScheduleInstallment extends AbstractAuditableWithUTCDa
     @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY, mappedBy = "installment")
     private Set<LoanInstallmentCharge> installmentCharges = new HashSet<>();
 
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER, mappedBy = "installment")
+    private Set<LoanTransactionToRepaymentScheduleMapping> loanTransactionToRepaymentScheduleMappings = new HashSet<>();
```

Comment:
I am a little hesitant whether we need to fetch the mapping eagerly. Can you tell me why did you decide to do so? Eager loading only required if we would read an association when the entity is already detached. I dont think that would happen here.

Reply:
> we would read an association when the entity is already detached.

Not necessarily. Eager fetching comes handy when the parent entity - from a business point of view - can't live without the child entity. 
I'm not sure if this is the case here though so I'm also waiting for some explanation here.

Reply:
Done, moved to LAZY